test/test_helper.rb

```diff
@@ -3,5 +3,10 @@
 require 'bundler/setup'
 require 'maxitest/autorun'
 require 'net/http'
-require 'webrick'
 require 'css_parser'
+require 'webmock/minitest'
+
+def fixture(*path)
+  Pathname(File.expand_path('fixtures', __dir__))
+    .join(*path)
+end
```

Comment:
needs to not be global or even the code could use it:
```suggestion
Minitest::Test.class_eval do
  def fixture(*path)
    Pathname(File.expand_path('fixtures', __dir__)).join(*path)
  end
end
```